resources/src/books.php

```diff
@@ -23,6 +25,9 @@ public function __construct()
       $this->ob = new ProjectConfig;
       $this->session = new SessionCookie;
    }
+   /*
+    * Function to display recent books
```

Comment:
Looks good as a starting point but also try to include documentation for parameters using in methods

Reply:
oooops right!